lib/Eclair/EIR/Lower.hs

```diff
@@ -289,13 +285,6 @@ codegenSymbolTable exts = do
   where
     intoIO = pure . runIdentity
 
--- TODO: add hash based on filepath of the file we're compiling?
-mkType :: Name -> [Table] -> ModuleBuilderT IO LLVM.Type
```

Comment:
This must've been a piece of code I forgot to delete during the addition of string support! 

Nice.